app/src/main/res/layout-land/profile_progress_header.xml

```diff
@@ -159,6 +159,7 @@
       android:text="@string/recently_played_stories"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="18sp"
+      android:visibility="@{viewModel.recentlyPlayedTopicCount > 0 ? View.VISIBLE : View.GONE, default=gone}"
```

Comment:
Could this be determined in the view model instead of having this if-check in the layout? This allows for easier changes in the future if the layout is changed.
So this could be `viewModel.textViewVisibility` (or `viewModel.getTextVisibility()`, etc.) & applied to the other layouts as well

Reply:
Done here and for other layouts.